package.json

```diff
@@ -5,6 +5,7 @@
   "main": "dist/index.js",
   "types": "dist/index.d.ts",
   "scripts": {
+    "generate-docs": "typedoc --mode file --readme none --plugin typedoc-plugin-markdown --hideSources --out docs src",
```

Comment:
It could be definitely nicer. Honestly, I have never implemented it. Always liked that documentation is nice and readable while one is in the text editor. HTML-files is less readable – however, the files produced by `typedoc` isn't that nice to read anyway. I'm torn, haha.

Are you thinking that we could have both? If so, maybe we can accept this change and test it in a separate branch?

Reply:
Having both would work, they could be generated by the same command:
``` 
"generate-docs": "typedoc --mode file --readme none --plugin typedoc-plugin-markdown --hideSources --out docs src &&  typedoc --mode file --readme none --plugin none --out docs src"
```

Or two separate: 
```
"generate-markdown": "typedoc --mode file --readme none --plugin typedoc-plugin-markdown --hideSources --out docs src",
"generate-html": "typedoc --mode file --readme none --plugin none --out docs src",
"generate-docs": "yarn generate-markdown && yarn generate-html"
```

Both is fine with me. But I agree that this can be done in a separate branch.

Reply:
Yes! Let's consider having two directories, I thought maybe there was a specialized Github Pages folder? Right?

Reply:
The specialized folder is actually `docs/` . Hm, I not sure how this could be done most gracefully.

Reply:
It is also possible to have the webpage-content on a separate branch, which could be automated with GitHub Actions: https://github.community/t/can-i-define-a-custom-source-or-folder-from-which-my-site-hosted-on-github-pages-can-load-from/10237/7 

If I got time this weekend I will give it a try, looks cool :-) Or if you want to.

Reply:
Fixed: https://mikaello.github.io/norwegian-national-id-validator/
It was only a minor change that was needed to the build description: https://github.com/mikaello/norwegian-national-id-validator/commit/17b866aeea576911999541bf4ac8de34ef275fc5